support-frontend/assets/components/checkoutForm/checkoutForm.scss

```diff
@@ -33,7 +33,12 @@ div .component-checkout-form-section.component-checkout-form-section--no-border
   border-top: none;
 }
 
+div .component-checkout-form-section.component-checkout-form-section--hidden {
+  display: none;
+}
```

Comment:
not sure if it's useful here, but in dotcom we used to have utility classes for things like hidden, meaning it was class `is-hidden` and it didn't depend on where it was.  I'm not sure if that's a proper BEM pattern but it did save having to reimplement hiding every time.
https://github.com/guardian/frontend/blob/840f5dbc0ebc03b606a75139d1b08bf949cb6fed/static/src/stylesheets/base/_helpers.scss#L87